pandas/tests/indexing/multiindex/test_setitem.py

```diff
@@ -210,6 +210,11 @@ def test_multiindex_assignment(self):
         with pytest.raises(ValueError, match=msg):
             df.loc[4, "c"] = [0]
 
+        # But with a length-1 listlike column indexer this works
+        df.loc[4, ["c"]] = [0]
```

Comment:
what happens with this case now?

Reply:
behaves like `df.loc[4, "c"] = 0`, just edited the comment to make that clear